lib/v2/onehu/radar.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    'onehu.xyz': {
+        _name: 'Onehu',
+        '.': [
+            {
+                title: '我是盐神',
```

Comment:
https://github.com/DIYgod/RSSHub/blob/eec19e594c95b89776b171831fda39ffa57f5d31/website/docs/routes/new-media.mdx#L5294
```suggestion
                title: '首页',
```